src/Services/privateApiService.js

```diff
@@ -42,4 +42,26 @@ export const getRequest =  async (url, id = null) => {
     }
 }
 
-
+export const patchApi = async( enpoint, id, data  ) => {
+    try {
+        const resp = await axios({
```

Comment:
Lo mismo, por temas de nomenclatura previa, debería llamarse response en vez de resp

```suggestion
        const response = await axios({
```